cmd/runtimetest/main.go

```diff
@@ -30,6 +32,13 @@ const PrGetNoNewPrivs = 39
 const specConfig = "config.json"
 
 var (
+	defaultFS = map[string]string{
+		"/proc":    "proc",
```

Comment:
https://github.com/opencontainers/runtime-spec/blame/master/config-linux.md#L15
In the `spec` is defined.